bundle/edu.gemini.wdba.xmlrpc.server/src/test/scala/edu/gemini/wdba/tcc/TcsConfigTest.scala

```diff
@@ -21,14 +21,6 @@ final class TcsConfigTest extends TestBase {
   }
 
   @Test
-  def testNonSiderealWithHorizonsDesignation(): Unit = {
```

Comment:
Is this test not needed anymore? It sounds like this change sounds should be accompanied by a test(s) case(s)